cosmos/carbon.json

```diff
@@ -197,6 +201,28 @@
         "high": 0.3692307692307693
       }
     },
+    {
+      "coinDenom": "EVMOS",
+      "coinMinimalDenom": "ibc/35E771B8682D828173F4B795F6C307780F96DC64D6F914FAE4CC9B4666F66364",
+      "coinDecimals": 18,
+      "coinGeckoId": "evmos",
+      "gasPriceStep": {
+        "low": 230769230769.23077,
+        "average": 288461538461.53846,
+        "high": 346153846153.84615
+      }
+    },
+    {
+      "coinDenom": "IRIS",
+      "coinMinimalDenom": "ibc/4E06CF24FEBFB3F5AF645377DCC0B70AA6183BAF6B918B8B6243FCDEB7D38118",
+      "coinDecimals": 6,
+      "coinGeckoId": "iris-network",
+      "gasPriceStep": {
+        "low": 0.4615384615384616,
+        "average": 0.5769230769230769,
+        "high": 0.6923076923076923
+      }
+    },
```

Comment:
@sarah-thong The IBC token cannot be used as a currency.
Please remove it.

If it is needed, it should be added to the chain as the native currency.

Reply:
Hi @HeesungB,

Duly noted, however I tried to add the IBC currencies (such as EVMOS and IRIS) into the native `currencies` list, I receive this error.
<img width="1098" alt="Screenshot 2023-10-09 at 10 13 10 AM" src="https://github.com/chainapsis/keplr-chain-registry/assets/65583522/fc26dd94-d7d3-4ac4-9d19-bb9f17e98e79">

Is there a way to circumvent this error? We would really appreciate it because we hope to add these IBC tokens as fee currencies to the chain info.

Reply:
I was wrong

You can add ibc tokens to FeeCurrencies.

But you can't add ibc tokens to Currencies.

Reply:
Okay noted, will make the necessary changes.

Reply:
@sarah-thong So Could you remove ibc token in `Currencies`field?

Reply:
@HeesungB ibc tokens removed from `currencies` field alr. Thanks for checking!